web/src/pages/index.md

```diff
@@ -7,36 +7,29 @@ title: ECE 477 Team 5 Homepage
-twist. The notable innovation is the incorporation of a responsive surface
-beneath a translucent playfield. This reactive surface, comprising strategically
-positioned lights, sensors, and computational components, reacts in real-time to
-the movements of the puck, generating visual displays that enhance the gameplay
-experience.
-
-The dynamic nature of the reactive surface introduces novel game modes that
-transcend the boundaries of conventional air hockey. Despite its technologically
-advanced features, the smart air hockey table maintains a compact form factor,
-making it adaptable to a variety of flat surfaces, from regular tables to other
-suitable platforms. This convergence of timeless gameplay with modern
-technological advancements is poised to redefine the enjoyment of air hockey,
-offering enthusiasts an immersive and unparalleled gaming encounter.
+The Smart Air Hockey Table replicates the functionalities of a traditional arcade air hockey setup with a twist. The notable innovation is the incorporation of a responsive surface beneath a translucent playfield. This reactive surface, comprising strategically positioned lights, sensors, and computational components, reacts in real-time to the movements of the puck, generating visual displays that enhance the gameplay experience.
+
+A tuned grid of Hall effect sensors lies beneath the playing field of the Smart Air Hockey Table. These sensors measure disturbances in the magnetic field caused by a magnet embedded within the puck. Microcontrollers within the table read the values generated by these Hall effect sensors and process the data to get an accurate position of the puck in real-time.
+
+A grid of RGB LEDs also lies beneath the playing field of the Smart Air Hockey Table. These RGB LEDs operate as an optional backlight for the playing field with variable colors and intensity. However, each RGB LED can be individually controlled, allowing for almost limitless complex visuals. When combined with puck tracking and the game state, these visuals become reactive to the gameplay.
+
+Beside these innovative features, the Smart Air Hockey Table retains all the typical features consumers expect from a recreational air hockey table designed for residential use. This includes robust goal detection using photoresistors and an LCD integrated into the table for displaying the current score and game state.
+
+The dynamic nature of the reactive surface introduces novel game modes that transcend the boundaries of conventional air hockey. Despite its technologically advanced features, the smart air hockey table maintains a compact form factor, making it adaptable to a variety of flat surfaces, from regular tables to other suitable platforms. This convergence of timeless gameplay with modern technological advancements is poised to redefine the enjoyment of air hockey, offering enthusiasts an immersive and unparalleled gaming encounter.
```

Comment:
Not sure what's the point of this paragraph. It's very confusing and I am not sure what it adds